pcweb/pages/docs/state/events.py

```diff
@@ -223,6 +265,20 @@ def events():
         doctext(
             "Setters are a great way to make your code more concise. But if you want to do something more complicated, you can always write your own function in the state."
         ),
+        subheader("Yielding multiple update"),
+        doctext(
+            "A regular event handler will send a StateUpdate when it has finished running.",
```

Comment:
Let's either do `pc.code(StateUpdate)` or call it "a state update".

Also - we need to add a space after each sentence or they will be joined together.